src/pages/publish-online-guidence/code snippets/GuidencePageExampleCodeSnippet.tsx

```diff
@@ -0,0 +1,174 @@
+export const GuidencePageExampleCodeSnippet = () => {
+  return (
+    <pre>
+      <code>
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;html&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;head&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;title&gt;</span>
+          My first HTML page
+          <span className="hljs-attr">&lt;/title&gt;</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-attr">&lt;link</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-name">rel</span>=
+          <span className="hljs-string">"stylesheet”</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-name">type</span>=
+          <span className="hljs-string">"text/css"</span>
+        </span>
+        <br />
+        &nbsp;&nbsp;&nbsp;&nbsp;
+        <span className="hljs-tag">
+          <span className="hljs-name">href</span>=
+          <span className="hljs-string">
+            ”https://cdn.statically.io/gh/digital-land/local-plans-timetable/master/timetable-visualisation.min.css”
```

Comment:
We'll need to make sure we're using the standard double quotes for the snippets:
![image](https://github.com/digital-land/local-plans-timetable/assets/29425751/9a012947-5a8c-4cbc-a257-6ef832d82c01)